test-cypress/integration/explore/graphs.overview.spec.js

```diff
@@ -1,114 +1,125 @@
-        "<urn:2> {<urn:a2> <urn:b2> <urn:c2> .}\n" +
-        "<urn:3> {<urn:a3> <urn:b3> <urn:c3> .}\n" +
-        "<urn:4> {<urn:a4> <urn:b4> <urn:c4> .}\n" +
-        "<urn:5> {<urn:a5> <urn:b5> <urn:c5> .}\n" +
-        "<urn:6> {<urn:a6> <urn:b6> <urn:c6> .}\n" +
-        "<urn:7> {<urn:a7> <urn:b7> <urn:c7> .}\n" +
-        "<urn:8> {<urn:a8> <urn:b8> <urn:c8> .}\n" +
-        "<urn:9> {<urn:a9> <urn:b9> <urn:c9> .}\n" +
-        "<urn:10> {<urn:a10> <urn:b10> <urn:c10> .}\n" +
-        "<urn:11> {<urn:a11> <urn:b11> <urn:c11> .}\n" +
-        "<urn:12> {<urn:a12> <urn:b12> <urn:c12> .}\n" +
-        "<urn:13> {<urn:a13> <urn:b13> <urn:c13> .}\n" +
-        "<urn:14> {<urn:a14> <urn:b14> <urn:c14> .}\n" +
-        "<urn:15> {<urn:a15> <urn:b15> <urn:c15> .}\n" +
-        "<urn:16> {<urn:a16> <urn:b16> <urn:c16> .}\n" +
-        "<urn:17> {<urn:a17> <urn:b17> <urn:c17> .}\n" +
-        "<urn:18> {<urn:a18> <urn:b18> <urn:c18> .}\n" +
-        "<urn:19> {<urn:a19> <urn:b19> <urn:c19> .}\n" +
-        "<urn:20> {<urn:a20> <urn:b20> <urn:c20> .}\n" +
-        "<urn:21> {<urn:a21> <urn:b21> <urn:c21> .}";
-
-
-    before(function () {
+const EXPORT_GRAPHS_TABLE_ID = '#export-graphs';
+
+describe('Graphs overview screen validation', () => {
+
+    let repositoryId;
+
+    let rdfTextSnippet1 = '<urn:1> {<urn:a1> <urn:b1> <urn:c1> .}\n' +
+        '<urn:2> {<urn:a2> <urn:b2> <urn:c2> .}\n' +
+        '<urn:3> {<urn:a3> <urn:b3> <urn:c3> .}\n' +
+        '<urn:4> {<urn:a4> <urn:b4> <urn:c4> .}\n' +
+        '<urn:5> {<urn:a5> <urn:b5> <urn:c5> .}\n' +
+        '<urn:6> {<urn:a6> <urn:b6> <urn:c6> .}\n' +
+        '<urn:7> {<urn:a7> <urn:b7> <urn:c7> .}\n' +
+        '<urn:8> {<urn:a8> <urn:b8> <urn:c8> .}\n' +
+        '<urn:9> {<urn:a9> <urn:b9> <urn:c9> .}\n' +
+        '<urn:10> {<urn:a10> <urn:b10> <urn:c10> .}\n' +
+        '<urn:11> {<urn:a11> <urn:b11> <urn:c11> .}\n' +
+        '<urn:12> {<urn:a12> <urn:b12> <urn:c12> .}\n' +
+        '<urn:13> {<urn:a13> <urn:b13> <urn:c13> .}\n' +
+        '<urn:14> {<urn:a14> <urn:b14> <urn:c14> .}\n' +
+        '<urn:15> {<urn:a15> <urn:b15> <urn:c15> .}\n' +
+        '<urn:16> {<urn:a16> <urn:b16> <urn:c16> .}\n' +
+        '<urn:17> {<urn:a17> <urn:b17> <urn:c17> .}\n' +
+        '<urn:18> {<urn:a18> <urn:b18> <urn:c18> .}\n' +
+        '<urn:19> {<urn:a19> <urn:b19> <urn:c19> .}\n' +
+        '<urn:20> {<urn:a20> <urn:b20> <urn:c20> .}\n' +
+        '<urn:21> {<urn:a21> <urn:b21> <urn:c21> .}';
+
+    beforeEach(() => {
+        repositoryId = 'repo' + Date.now();
+        cy.createRepository({id: repositoryId});
+
         cy.visit('/repository');
-        // Create new one
-        cy.createNewRepo(repoId).wait(2000);
-        cy.setRepoDefault(repoId);
-        cy.selectRepo(repoId);
+        cy.setRepoDefault(repositoryId);
+        cy.selectRepo(repositoryId);
 
-        // Import Wine dataset which will be used in most of the following tests
```

Comment:
@mihailradkov  I'll refactor this to use the rest in another PR once you merge your PR.